pkg/windows/windows.go

```diff
@@ -562,6 +562,11 @@ func (vm *windows) runBootstrapper() error {
 	if vm.clusterDNS != "" {
 		wmcbInitializeCmd += " --cluster-dns " + vm.clusterDNS
 	}
+	// check log level and increase kubelet verbosity accordingly
```

Comment:
Please add a new line before the comment to be consistent with the code in master